node/tessel-export.js

```diff
@@ -916,6 +916,10 @@ Tessel.LED.prototype.toggle = function(callback) {
   this.write(this.value ? 0 : 1, callback);
 };
 
+Tessel.LED.prototype.isOn = function() {
+  return this.value === 1;
+};
+
```

Comment:
>  So should this PR be closed?

No way! Let's wait for @johnnyman727 to respond to my counter argument. It's either "land as is" or you'll make another commit on your branch and push the changes—this PR will be automatically updated :)